src/Integration/Resources/Strings.resx

```diff
@@ -772,4 +772,16 @@ This is the general format to use when writing errors to output window or when t
   <data name="ExclusionGetError" xml:space="preserve">
     <value>Error loading server settings. Analysis settings from the server (e.g. inclusions/exclusions) will not be applied in the IDE. Error: {0}</value>
   </data>
+  <data name="ExclusionSaveError" xml:space="preserve">
+    <value>Error writing settings for project. File exclusions on the server may not be excluded in the IDE. Error: {0}</value>
+  </data>
+  <data name="ExclusionOnStandaloneNotSupported" xml:space="preserve">
```

Comment:
Where is this string used? (the PR is changing the binding code, and I can't see a reason why the binding code would ever need such a string).

Reply:
This was used in the Get which is a valid scenario. While merging with Rita's changes it became orphan.